programs/biome.nix

```diff
@@ -11,6 +10,10 @@ let
   t = l.types;
   p = pkgs;
 
+  cfg = config.programs.biome;
+  biomeVersion = if builtins.match "^1\\." pkgs.biome.version != null then "1.9.4" else "2.1.2";
```

Comment:
A couple thoughts:

1. I am not really understanding this ternary. How does 1.x equate to 1.9.4, and how does "not 1.x" equate to 2.1.2? This either needs reworking, or at least a comment.
2. The `if biomeVersion == "1.9.4" then` down below should get moved closer to this code. It feels very related, but is so far away it'll be more likely that things will get out of whack.